cli/network.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/alibaba/pouch/apis/types"
+
+	"github.com/spf13/cobra"
+)
+
+// NetworkCommand is used to implement 'network' command.
+type NetworkCommand struct {
+	baseCommand
+}
+
+// Init initializes NetworkCommand command.
+func (n *NetworkCommand) Init(c *Cli) {
+	n.cli = c
+
+	n.cmd = &cobra.Command{
+		Use:   "network [command]",
+		Short: "Manage pouch networks",
```

Comment:
Please add Long and Example for new initialized Command, like https://github.com/alibaba/pouch/blob/master/cli/create.go#L28-L33.
They could help a lot in doc generation.